src/shared/sam/debugger/awsSamDebugConfigurationProvider.ts

```diff
@@ -1,70 +0,0 @@
-/*!
```

Comment:
this module is redundant with `src/shared/sam/debugger/awsSamDebugger.ts`

Reply:
I was thinking it'd be best to remove `awsSamDebugger` instead of this file. Interested to head @ckant 's thoughts

Reply:
Why are there two vscode.DebugConfigurationProvider implementations? This seems like an accident somehow.
What is the difference in the two implementations?

Reply:
there's no difference, this is just merge cleanup.